README.md

```diff
@@ -19,53 +19,105 @@ how it works in [this blog post](http://blog.humphd.org/thimble-and-bramble/).
+* Run ``git submodule update --init`` to install submodules
+* Run ``npm install`` to install dependencies
+* Run `npm run build` to build Bramble
+* Run ``npm start`` to get a static server running on [http://localhost:8000/src](http://localhost:8000/src). You can try the demo version at [http://localhost:8000/src/hosted.html](http://localhost:8000/src/hosted.html)
 
-### Editor
+## 2) Thimble Installation
+### Automated Installation (Preferred Method)
+After forking and cloning Thimble, installing it is quite simple. All we need to do is run the following command inside the cloned directory and let vagrant do the heavy lifting for us:
 
-To fully use Thimble locally, you need to first setup Brackets locally first. This can be done by following the steps outlined below:
+```
+vagrant up
+```
 
-- Fork the [Brackets repository](https://github.com/mozilla/brackets) and then clone it to your local machine using `git clone --recursive https://github.com/<your_username>/brackets.git` (replace `<your_username>` with your Github username for the account you forked Brackets into)
-- In the cloned repository directory, run `npm install` to install the dependencies for Brackets
-- Run `npm run build` to create the built editor files that will be used by Thimble
-- Run `npm start` to start a server that will allow the editor to be accessed on [http://localhost:8000/src](http://localhost:8000/src)
-- You can find out more information about setting up Brackets locally by referring to the instructions [here](https://github.com/mozilla/brackets#how-to-setup-bramble-brackets-in-your-local-machine)
+Depending on your internet connection speed, this process can take a while (Since it needs to download all dependencies not listed above).
+Once you see `Express server listening on http://localhost:3500`, you are ready to start using Thimble!
+Any changes made to the Thimble source code on your system will automatically be reflected on [http://localhost:3500](http://localhost:3500).
 
-### Thimble and Services
+To stop Thimble, simply press `Ctrl+C` twice.
 
-Thimble interacts with the Publish API (source managed in [publish.webmaker.org](https://github.com/mozilla/publish.webmaker.org)) to store users, projects, files and other content as well as publish user projects.
+To restart Thimble, run:
 
-For authentication and user management, Thimble uses Webmaker OAuth which consists of the Webmaker ID System (source managed in [id.webmaker.org](htps://github.com/mozilla/id.webmaker.org)) and the Webmaker Login API (source managed in [login.webmaker.org](https://github.com/mozilla/login.webmaker.org)).
+```
+vagrant reload --provision
+```
 
-All three services along with Thimble are bundled together using Git subtrees and run together using Vagrant.
+This will take less time to setup compared to the `vagrant up` command.
 
-The first step is to fork and clone Thimble and navigate to the cloned directory in a terminal shell.
+### Manual Installation (Optional)
```

Comment:
@gideonthomas Then we disagree. Remember that this README is also for tomorrow's user, including people well version in development, who can't use vagrant due to Hyper-V requirements on windows. They should not have to hunt for instructions, and having a "more detailed section" at the bottom of the immediate project documentation is quite common for larger projects (Which Thimble definitely falls under). I would very much advocate we keep the manual installation instructions immediately available rather than locked away in a "you need to go back online" (the README.md downloads with thimble as part of a `git clone` operation - the wiki most definitely does not)